controllers/storagecluster/storageclasses.go

```diff
@@ -231,6 +245,38 @@ func newCephBlockPoolStorageClassConfiguration(initData *ocsv1.StorageCluster) S
 	}
 }
 
+// newCephNFSStorageClassConfiguration generates configuration options for a Ceph Filesystem StorageClass.
+func newCephNFSStorageClassConfiguration(initData *ocsv1.StorageCluster) StorageClassConfiguration {
+	persistentVolumeReclaimDelete := corev1.PersistentVolumeReclaimDelete
+	// VolumeExpansion is not yet supported for nfs.
+	allowVolumeExpansion := false
+	return StorageClassConfiguration{
+		storageClass: &storagev1.StorageClass{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: generateNameForCephNetworkFilesystemSC(initData),
+				Annotations: map[string]string{
+					"description": "Provides RWO and RWX Filesystem volumes",
+				},
+			},
+			Provisioner:          fmt.Sprintf("%s.nfs.csi.ceph.com", initData.Namespace),
```

Comment:
Why is there no `generateNameForNFSCSIProvisioner()` (or similar) here?

Reply:
done